apps/web/src/common/modules/modals/notification-email-modal/NotificationEmailModal.vue

```diff
@@ -1,40 +1,52 @@
 <template>
     <p-button-modal
+        v-if="state.proxyModalType === MODAL_TYPE.SEND"
```

Comment:
suggestion: The code is quite long. How about separating 'Enter Notification Email' and 'Verify Notification Email' into different files? Just suggestion..not important 😅